src/content/topics/integrations/release.mdx

```diff
@@ -0,0 +1,120 @@
+
+* **Project Key**: This is the key for the project where you wish to create environments. The default value is _default_ , because LaunchDarkly always has a project with key _default_.
+* **Access Token**: This is the token for the account you would like Release to create environments under in LaunchDarkly. You can create or find the access token you want to use in LaunchDarkly in the **Authorization** tab of the **Account Settings** page. All access tokens start with **api-**.
+* **Environment Variables**: The last three form elements define the names of the environment variables where Release stores the relevant API keys from LaunchDarkly. You can change them to anything you like. You may need the LaunchDarkly access token to set up your SDK client to choose flag variations in your application. After you set up the environment in LaunchDarkly, Release stores the various API keys values in these environment variables. This gives your container access to the API keys you need. They are stored as encrypted Kubernetes secrets.
+
+![An example LaunchDarkly configuration.](../../images/release-setup-ld-integration.png)
+
+After you save the configuration, Release attempts to use your **Access Token** to connect to your LaunchDarkly account. If that works, you're done. The integration is ready to go. If not, errors appear for you to troubleshoot.
+
+## Using LaunchDarkly with the Release integration
+
+The integration consists of two parts, both of which are automated and handled by Release.
+
+* **[Pre-Deployment step](#using-the-pre---deployment-step):** In this step, the integration adds a task to each of your deployments to create the feature flag environment.
+* **[Pre-Delete step](#using-the-pre---delete-step):** When you delete an environment, this step deletes the environment in LaunchDarkly.
+
+![The ephemeral environment creation workflow](../../images/release-ld-integration-workflow.png)
+
+### Using the Pre-Deployment step
+
+The Pre-Deployment step is for when you first deploy a space or deploy a new configuration. It only creates the environment in LaunchDarkly the first time it runs.
+
+![Environment &apos;ted422c&apos; was created in LaunchDarkly.](../../images/release-pre-deployment-step.png)
+
+After a successful creation, the environment variables you specified during set up populate and are exposed to your containers with Kubernetes secrets.
+
+![LaunchDarkly-specific environments as Kubernetes secrets.](../../images/release-ld-envs.png)
+
+You can use these environment variables in your code to access your feature flags. 
+
+### Using the Pre-Delete step
+
+The Pre-Delete step is invoked anytime a Release environment is removed, either manually through the UI or after a pull request is merged or closed. The integration removes the corresponding LaunchDarkly environment automatically when the Release environment is removed.
+
+## Example: Setting up the LaunchDarkly SDK client in Rails
```

Comment:
I'm still a bit confused with the examples. Perhaps this is just due to my lack of understanding of how the Release+LaunchDarkly integration works.

Right now the examples just show how to use LaunchDarkly. They don't show any interaction with Release. These examples could just as well be for any LaunchDarkly usage. Now perhaps that's your point -- that customers can use LaunchDarkly just like they would normally, all they need is the SDK key or API key to interact with it.

However, if that's the case, we can be clearer by replacing the entire example sections with:

```
You can now use your [LaunchDarkly SDK](https://docs.launchdarkly.com/sdk) as you normally would.
```

I think that examples of how to use the LaunchDarkly SDKs are very appropriate when in the context of [your documentation](https://docs.releaseapp.io/integrations/integrations-overview/launchdarkly-integration#example-setting-up-launchdarkly-sdk-client-in-rails), but less so in our documentation since customers likely already know how to use our SDKs.

Reply:
That's true,  I mostly just wanted to show that when the integration is all said and done, you interact with it using Environment Variables.  The reason for the example is that not everyone uses ENVs so I didn't want them to just get to the end and be like...ok, so how do i use this thing.  I was about to just remove the example when i submitted the first time.

As i type this I think you are right.  The example will be found in our docs and we will have more soon, using reactjs, etc.  So if they were using the integration they could just look at our docs!

Reply:
Are environment variables important because....they're also used by Release? Or because they're set by Release?

If the key is that they're using environment variables, let's just say that without the Ruby-specific example. That'll be more to the crux of what we want users to know when starting to use the integration.

Reply:
Would you mind if i include a link at the bottom to our docs?  It would say something like, "See examples over at releaseapp.io.", and link to our docs.

Reply:
That's perfectly fine 👍